pkgs/development/python-modules/srp/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchPypi, six, lib }:
+
+buildPythonPackage rec {
+  pname = "srp";
+  version = "1.0.15";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "d5b8ed6dc7d3ae2845a16590ef37763bbf15d6049848b85a8c96dfb8a83c984a";
+  };
+
+  propagatedBuildInputs = [ six ];
+
+  # Package doesn't distribute tests.
+  doCheck = false;
+
+  meta = with stdenv.lib; {
```

you import lib anyawy
```suggestion
  meta = with lib; {
```